grpclog/loggerv2.go

```diff
@@ -123,16 +123,37 @@ type loggerV2Config struct {
 }
 
 func newLoggerV2WithConfig(infoW, warningW, errorW io.Writer, c loggerV2Config) LoggerV2 {
-	var m []*log.Logger
 	flag := log.LstdFlags
 	if c.jsonFormat {
 		flag = 0
 	}
-	m = append(m, log.New(infoW, "", flag))
-	m = append(m, log.New(io.MultiWriter(infoW, warningW), "", flag))
-	ew := io.MultiWriter(infoW, warningW, errorW) // ew will be used for error and fatal.
-	m = append(m, log.New(ew, "", flag))
-	m = append(m, log.New(ew, "", flag))
+
+	infoLogW := infoW
+
+	// use io.Discard instead of io.MultiWriter if all loggers at a given level are io.Discard.
```

Comment:
nit: Please keep the comment lines upto 80 columns. Thanks

Reply:
Changed